nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryQuery.java

```diff
@@ -2096,6 +2096,34 @@ public void testOrderByAndDistinct() throws Exception {
         assertIdSet(dml, file1.getId());
     }
 
+    @Test
+    public void testWildCardsInFulltextSearch() throws Exception {
+        createDocs();
+
+        String query;
+        DocumentModelList dml;
+
+        DocumentModel file1 = session.getDocument(new PathRef("/testfolder1/testfile1"));
+        file1.setProperty("dublincore", "title", "hello world 1");
+
+        session.saveDocument(file1);
+        session.save();
+
+        waitForFulltextIndexing();
+
+        query = "SELECT * FROM File Where ecm:fulltext = 'hello world 1%'";
```

Comment:
Please use `*` instead of `%` in fulltext queries, even in tests. `%` is a historical compatibility character.